api/get/api_get.py

```diff
@@ -158,8 +143,23 @@ def get_ride_info_for_specific_day():
 			)
 			if last_24_hours_rides:
 				rides = [tuple_to_dict(entry, 'rides') for entry in last_24_hours_rides]
-				return jsonify(rides)
+				return rides
 			else:
-				return jsonify({"output": 'No matching records'})
+				return {"output": 'No matching records'}
 	except Exception as e:
-		return jsonify({'error': str(e)})
+		return {'error': str(e)}
+
+def lambda_handler(event, context):
```

Comment:
this is really neat and tidy! Like that the lambda_handler is so small with all the functionality outsourced to other functions!